fonts.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
+    font-family: 'pt_serif' serif;
+    src: url('fonts/ptserif-bold-webfont.woff2') format('woff2'),
+         url('fonts/ptserif-bold-webfont.woff') format('woff');
+    font-weight: bold;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6170621)